src/qibo_client/qibo_client.py

```diff
@@ -68,8 +68,7 @@ def run_circuit(
         self,
         circuit: qibo.Circuit,
         nshots: int = 1000,
-        lab_location: str = "tii",
-        device: str = "sim",
+        device: str = "k2",
```

Comment:
I would recommend to do not set a default device at all, the user must specify shots (if any for simulation) and device.

Reply:
Let me postpone this discussion in a dedicated issue.